demos/python_demos/bert_question_answering_demo/bert_question_answering_demo.py

```diff
@@ -157,10 +158,19 @@ def main():
     log.info("Loading model to the {}".format(args.device))
     ie_encoder_exec = ie.load_network(network=ie_encoder, device_name=args.device)
 
-    # loop on user's questions
-    while True:
-        question = input('Type question (empty string to exit):')
-        if not question:
+    if args.questions:
+        def questions():
+            for question in args.questions:
+                log.info("Question: {}".format(question))
+                yield question
+    else:
+        def questions():
+            while True:
+                yield input('Type question (empty string to exit):')
+
+    # loop on user's or prepared questions
+    for question in questions():
+        if not question or len(question.strip()) == 0:
```

Comment:
```suggestion
        if not question.strip():
```

Reply:
The same pattern is also used in the other demos.

Reply:
Done